### ISSUE TITLE
Navigation landmarks need accessible names.

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3514]
**Issue Description**: The sidebar areas are both navigation landmarks. Since there are two, they need unique descriptive names to distinguish them from one another. 

Related to All page content is not in landmarks #3515

**WCAG Criteria**: [4.1.2 Name, Role, Value - A](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value)

**Screenshot or Snippet**: [Link to Screenshot](https://share.zight.com/QwuoOqWx)

**Notes**: Possible name could be "Chat history" for the sidebar navigation on the left. The other could be "Chat controls."  The [navigation](https://w3c.github.io/aria/#navigation) is also not the correct role for this sidebar. It could be better described its own [region](https://www.w3.org/TR/wai-aria/#region). As a region landmark it should still have an accessible name.
<img width="1617" alt="Screenshot 2024-08-01 at 2 54 10 PM" src="https://github.com/user-attachments/assets/21c08add-0a03-4fa6-bbb0-f59042b39343" style="border:solid 1px black">

**Comment by derekjackson-das** [comment]
I think we can close this one and we can focus on the landmark roles and naming in #3515
